SoD_SP_Public/Next Version/SoD_SP.ini

```diff
@@ -94,37 +94,37 @@ BattleCoord = Black
 # Es deaktiviert „den Monat der SEUCHE!“
 # Απενεργοποιεί «τον μήνα της ΠΑΝΟΥΚΛΑΣ!»
 # Wyłącza "miesiąc PLAGI!".
-DisablePlague = 1
+DisablePlague = 0
```

Comment:
The default ini options should not be modified, just the comments explaining what they do.